src/main/java/net/comments/objects/CommentTable.java

```diff
@@ -0,0 +1,10 @@
+package net.comments.objects;
+
+import java.util.List;
+
+public interface CommentTable {
+    void selectComment(int commentsNumber);
+    void setCommentsTable();
+    List<BComment> getCommentsTable();
```

Comment:
`List<BComment> comments = new CommentTable(driver).getCommentsTable()` - look not clear. It'll be greate rename `getCommentsTable` to `comments` or something similar.

Reply:
I've renamed it to getTableComments